Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+group :test do
+  gem 'rspec'
```

Comment:
but the current test suite runs equally well on both, right? not sure if it's worth locking it to 2.x or 3.x
(also this decision should not take logstash into account)